Makefile

```diff
@@ -10,11 +10,14 @@ all: clean test
 
 clean:
 	$(PYTHON) setup.py clean
-	rm -rf dist
+	rm -rf dist build bin
 
+bin:
+	mkdir -p $@
+	PYTHONPATH=bin:$(PYTHONPATH) python setup.py develop --install-dir $@
 
-test-code:
-	$(NOSETESTS) -s -v $(MODULE)
+test-code: bin
+	PATH=bin:$(PATH) PYTHONPATH=bin:$(PYTHONPATH) $(NOSETESTS) -s -v $(MODULE)
```

Comment:
I thought to add it to our buildbot rules, but didn't work out there either for some reason:

http://smaug.datalad.org:8020/builders/datalad-pr-docker-dl-nd80/builds/227/steps/test%20%27make%20test%27/logs/stdio

Lazy/busy to deal with it ATM but something to keep in mind (I will revert buildbot to original shape)